tests/manager/api/controller/test_urn_lookup.py

```diff
@@ -12,7 +12,10 @@ def test_work_lookup(self, controller_fixture: ControllerFixture):
         work = controller_fixture.db.work(with_open_access_download=True)
         [pool] = work.license_pools
         urn = pool.identifier.urn
-        with controller_fixture.request_context_with_library("/?urn=%s" % urn):
+        with (
+            controller_fixture.request_context_with_library("/?urn=%s" % urn),
+            controller_fixture.wired_container(),
```

Comment:
This is needed because the containers must be setup now in order to get the registry.